CheckManga/checkmanga.py

```diff
@@ -0,0 +1,128 @@
+import requests
+import sqlite3
+import bs4 as bs
+import os
+from flask import Flask, render_template, make_response, jsonify, g
+
+
+app = Flask(__name__)
+
+app.config.from_object(__name__)
+app.config.update(dict(
+    DATABASE=os.path.join(app.root_path, 'mycheckmanga.db'),
+    SECRET_KEY='development key',
+    DEBUG=True,
+    #USERNAME='admin',
+    #PASSWORD='default'
+))
+
+@app.route('/')
+def main_page():
+    """Returns the main page for the check_manga site."""
+    return render_template('main_page.html')
+
+@app.route('/checkmanga/', methods=['POST'])
```

Comment:
maybe read up how RESTful API endpoints are named - they are generally objects (manga, title, site) not verbs

Reply:
hmmm I see will look into this further